src/mixins/bcol-mixin.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Vue } from 'vue-property-decorator'
+import axios from '@/axios-auth'
+
+@Component({})
+export default class BcolMixin extends Vue {
+  async getErrorObj (errCode) {
+    const fetchUrl = sessionStorage.getItem('PAY_API_URL') + 'codes/errors/' + errCode
```

Comment:
The Pay API URL should be a getter so you don't have to access session storage every time. (Getters are reactive and cached.)

Reply:
done

Reply:
done

Reply:
Fixed.